testsuite/src/it/java/io/quarkus/ts/startstop/ArtifactGeneratorTest.java

```diff
@@ -107,13 +107,13 @@
             "resteasy-jsonb",
             "scheduler",
             "spring-boot-properties",
-            "smallrye-reactive-messaging-amqp",
+            "smallrye-reactive-streams-operators",
             "spring-data-jpa",
             "spring-di",
             "spring-security",
             "spring-web",
             "undertow",
-            "undertow-websockets",
```

Comment:
Still available, let's keep it here for 1.x -> 2.y purposes

https://github.com/quarkusio/quarkus/blob/main/extensions/websockets/legacy/pom.xml

Reply:
We can't use it because it was removed from the descriptor: https://github.com/quarkusio/quarkus/blob/9dd2696d02156164601d6f395f23e28478a300a1/devtools/platform-descriptor-json/src/test/resources/fakeextensions.json, therefore it's not found when using -Dextensions=undertow-websockets

Reply:
I''ve not done this change but will keep this PR open until this is clarified.

Reply:
ok, so let's drop `"undertow-websockets"`